contrib/zmq/zmq_sub.py

```diff
@@ -69,6 +68,12 @@ async def handle(self) :
         elif topic == b"rawtx":
             print('- RAW TX ('+sequence+') -')
             print(binascii.hexlify(body))
+        elif topic == b"sequence":
+            hash = body[:32].hex()
```

Comment:
Use bytes instead of string
```suggestion
            hash = binascii.hexlify(body[:32])
```